internal/pkg/bccsp/factory/test-plugin/plugin.go

```diff
@@ -8,7 +8,7 @@ package main
 import (
 	"hash"
 
-	"github.com/hyperledger/fabric/bccsp"
+	"github.com/hyperledger/fabric-ca/internal/pkg/bccsp"
```

Comment:
This is going to be problematic for plugins.

Reply:
Ah ... right ... I moved it into internal in master but it needs to be external here.

Reply:
will move this under pkg/bccsp for now